From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 16 Dec 2008 21:26:07 +0000 (-0800)
Subject: USB: option: increase outgoing buffer size and number
X-Git-Tag: v2.6.29-rc1~182^2~61
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Ftorvalds%2Flinux-2.6.git;a=commitdiff_plain;h=f1632df36b9467b75b7abfd2799aef67ec74a60a

USB: option: increase outgoing buffer size and number

This should speed up the option driver's upload speed quite a bit.  It has been tested by a number of different people on different devices with success.

Cc: Roland Wolters <roland.wolters@credativ.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

diff --git a/drivers/usb/serial/option.c b/drivers/usb/serial/option.c
index 4c9497d..5ed1834 100644
--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -522,9 +522,9 @@ static int debug;
 /* per port private data */
 
 #define N_IN_URB 4
-#define N_OUT_URB 1
+#define N_OUT_URB 4
 #define IN_BUFLEN 4096
-#define OUT_BUFLEN 128
+#define OUT_BUFLEN 4096
 
 struct option_port_private {
 	/* Input endpoints and buffer for this port */
